manifests/charts/gateways/istio-ingress/templates/preconfigured.yaml

```diff
@@ -46,7 +46,8 @@ spec:
       operation: INSERT_AFTER
       value:
         name: "envoy.filters.network.tcp_cluster_rewrite"
-        config:
+        typed_config:
```

Comment:
@nmittler what is the long term plan for this? We clearly don't have a test - are we planning to keep this around?

Reply:
I'm actually not familiar with this ... is it needed for the `.global` configuration?

Reply:
I guess the ingress needs this to make `.global` work? I'd be concerned about getting rid of this until folks are migrated over to the `cluster.local` way of doing things. We don't even have the new docs published yet.

Reply:
> I guess the ingress needs this to make .global work? 

As far as I know, yes. Currently, we are using a setup like this:

```
+-----------------------------------------+
|MESH A                                   |
|                                         |
|    +----------------------+             |
|    |serviceentry          |             |
|    |                      |             |
|    |httpbin.demo.global   |             |
|    +----------+-----------+             |
|               |                         |
+-----------------------------------------+
                |
+-----------------------------------------+
|MESH B         |                         |
|               v                         |
|   +-----------+---------------------+   |
|   |ingress mtls gateway             |   |
|   +---------------------------------+   |
|   |envoyfilter                      |   |
|   |                                 |   |
|   |s/\.global$/.svc.cluster.local/  |   |
|   +-----------+---------------------+   |
|               |                         |
|               |                         |
|   +-----------v---------------------+   |
|   |service                          |   |
|   |                                 |   |
|   |httbin.demo                      |   |
|   +---------------------------------+   |
+-----------------------------------------+
```

Reply:
Without this fix I could not get Multicluster networking to work in 1.7.0+
All works correctly for me after I adjust the yaml as per the change detailed here.

Reply:
Hi guys, any progress here? In 1.7+ Multicluster doesn't work according to the docs. Is it not a serious issue? 

@nmittler Can you maybe explain what is "the `cluster.local` way of doing things" you have mentioned ^? Or where can we read about that?

Reply:
the new installation docs are here: https://preliminary.istio.io/latest/docs/setup/install/multicluster/